README.rst

```diff
@@ -1,6 +1,6 @@
-========
 envparse
 ========
+``envparse`` is a simple utility to help parsing environment variables.
```

Comment:
Minor grammatical error here; should be "envparse is a simple utility to parse environment variables".